### DISCUSSION TITLE
Feature flags to actually be able to disable the AI improvements - to  make the game easier / and in spirit of original

### DISCUSSION THREAD

**Discussion by came-here-to-learn** [ihhub/fheroes2#8291] [Ideas]
Do the new changes make a monster out of AI?

Also: can you cut the weekly growths of random map monsters, those are incredible.

The campaign also feels unbeatable when the AI is not incredibly stupid.

**Comment by zenseii** [comment]
Hi, @came-here-to-learn and thank you for checking out the project!

There are no changes made to the original AI in fheroes2, because the fheroes2 AI is coded from scratch, which means we can't just switch off things.

As for your concerns about the AI being too tough I suggest to try it and see, since from your post it seems like you haven't tried fheroes2 yet.

For experienced players the AI should be quite manageable to beat. We are also regularly checking whether the AI is approachable on lower difficulties or not, there's a significant change in this regard being worked on right now.

The weekly growth might be modifiable in the new map maker in the future but it will not be highly prioritized. We have a policy on adding options into the game that adding such an option wouldn't really fit into.
